app/assets/stylesheets/modules/forms.scss

```diff
@@ -65,6 +65,11 @@ hr {
   white-space: nowrap;
 }
 
+.single-item-callnumber {
+  margin: -1em 0 0.5em;
+  color: #8c1515;
```

Comment:
Please make this a variable and put it in https://github.com/sul-dlss/sul-requests/blob/main/app/assets/stylesheets/sul-variables.scss#L1